build/root/usr/local/bin/ci_entrypoint_gpu-operator.sh

```diff
@@ -127,6 +193,10 @@ shift
 set -x
 
 case ${action} in
+    "test_upgrade_commit")
+        test_upgrade_commit "https://github.com/NVIDIA/gpu-operator.git" master
+        exit 0
+        ;;
```

Comment:
one thing I had in mind for testing such PRs that are "out of the main test paths" would be to add an entry in [`run`](https://github.com/openshift-psap/ci-artifacts/blob/master/build/root/usr/local/bin/run):
```
case ${target} in
  "custom")
     ci_entrypoint_custom.sh
     exit 0
     ;;
esac
```

and in `ci_entrypoint_custom.sh` you could test whatever you want, ie, here, call `ci_entrypoint_gpu-operator.sh test_upgrade_commit`

and in `openshift-release`, we add a `\test custom` to trigger this execution path. WDYT?

ideally `ci_entrypoint_custom.sh` would be removed by a GitHub Actions postcommit hook, but that would require a bit of git/github deep dive :)

Reply:
Sounds good. I guess after the tests pass, as long as the final force push removes just the call from the custom hook, then we can merge it untested

Reply:
yes, the postcommit hook would allow a "stronger" review flow, as you can `/lgtm`, and the hook would take care of removing the content of the file

but removing before merging is good as well, not like we've for new entrypoint tests everyday ;-)

Reply:
https://github.com/openshift-psap/ci-artifacts/pull/239
https://github.com/openshift/release/pull/21026